client/src/pages/JoinPage/style.scss

```diff
@@ -35,4 +35,14 @@
       }
     }
   }
+
+  .password-visibility-button {
+    $eye-width: 1rem;
+
+    background: $no-background-color;
+    color: $gray;
+    width: $eye-width;
+    margin-left: calc($eye-width * -2 - $spacing);
+    margin-right: $spacing;
```

Comment:
Yes, we are getting out of atomize, but let's use `suffix` property of atomize Input for this to avoid unintuitive `margins`.

https://atomizecode.com/docs/react/atoms/

![image](https://user-images.githubusercontent.com/9146455/198401462-885da1b6-f535-4a5c-93fe-a7af3a1af1d7.png)

Reply:
Since I still wanted to stay away from atomize as much as possible, I tried doing
```
position: relative;
```
on the wrapper and
```
position: absolute;
right: 0.5rem;
```
for the button itself, hopefully it's a lot clearer now.